autogpt_platform/frontend/src/app/(platform)/marketplace/components/MainCreatorPage/MainCreatorPage.tsx

```diff
@@ -39,7 +39,7 @@ export const MainCreatorPage = ({ params }: MainCreatorPageProps) => {
 
   if (creator)
     return (
-      <div className="mx-auto w-screen max-w-[1360px]">
+      <div className="mx-auto w-full max-w-[1360px]">
```

Comment:
There was horizontal overflow on the pages....

<img width="1841" height="862" alt="Image" src="https://github.com/user-attachments/assets/9716ce68-7234-4b5f-89b7-db5926ae7fc2" />